plenigo_i18n/plenigo-de_DE.po

```diff
@@ -1,688 +1,658 @@
+msgstr "Verwenden plenigo Authentifizierungsanbieter"
 
-#: plenigo_plugin/PlenigoShortcodeManager.php:390
-msgid "Customer ID:"
-msgstr "Kundennummer:"
+#: ../plenigo_plugin/settings/SettingWelcomeURL.php:54
+msgid "URL After NEW USER"
+msgstr "URL Nach BENUTZER ANMELDUNG"
 
-#: plenigo_plugin/PlenigoShortcodeManager.php:391
-msgid "Email Address:"
-msgstr "E-Mail:"
+#: ../plenigo_plugin/settings/SettingWelcomeURL.php:62 ..
+#: plenigo_plugin/settings/SettingLoginURL.php:62
+msgid "Leave empty for no redirection..."
+msgstr "Leer lassen für keine Umleitung ..."
 
-#: plenigo_plugin/PlenigoShortcodeManager.php:392
-msgid "User Name:"
-msgstr "Benutzername:"
+#: ../plenigo_plugin/settings/SettingDebugMode.php:61
+msgid "Debug Mode"
+msgstr "Debug-Modus"
 
-#: plenigo_plugin/PlenigoShortcodeManager.php:393
-msgid "Gender:"
-msgstr "Anrede:"
+#: ../plenigo_plugin/settings/SettingDebugMode.php:90
+msgid "NO DEBUG mode"
+msgstr "Keine Debug-Modus"
 
-#: plenigo_plugin/PlenigoShortcodeManager.php:394
-msgid "Full Name:"
-msgstr "Name:"
+#: ../plenigo_plugin/settings/SettingDebugMode.php:92
+msgid "VERBOSE DEBUG mode"
+msgstr "Ausführliche Debugmodus"
 
-#: plenigo_plugin/PlenigoShortcodeManager.php:395
-msgid "First Name:"
-msgstr "Vorname."
+#: ../plenigo_plugin/settings/SettingDebugMode.php:94
+msgid "VERBOSE LOG FILE mode. Up to: "
+msgstr "Ausführliche Protokolldatei-Modus. Bis zu:"
 
-#: plenigo_plugin/PlenigoShortcodeManager.php:396
-msgid "Last Name:"
-msgstr "Nachname:"
+#: ../plenigo_plugin/settings/SettingWooProductType.php:65
+msgid "Product Type"
+msgstr "Produktart"
 
-#: plenigo_plugin/PlenigoShortcodeManager.php:397
-msgid "Street Address:"
-msgstr "Straße und Nummer:"
+#: ../plenigo_plugin/settings/SettingPreventTag.php:70
+msgid "Prevent Payment Tag"
+msgstr "Verhindern Zahlungs Etikett"
 
-#: plenigo_plugin/PlenigoShortcodeManager.php:398
-msgid "Additional Information:"
-msgstr "Zusatzinformation:"
+#: ../plenigo_plugin/settings/SettingCurtainBuyTextDB.php:70
+msgid "Buy Button Text (based on tag)"
+msgstr "Kaufen Button-Text (basierend auf den Etikett)"
 
-#: plenigo_plugin/PlenigoShortcodeManager.php:399
-msgid "ZIP Code:"
-msgstr "PLZ:"
+#: ../plenigo_plugin/settings/SettingCurtainBuyTextDB.php:81
+msgid "Buy button text..."
+msgstr "Kaufen Button text ..."
 
-#: plenigo_plugin/PlenigoShortcodeManager.php:400
-msgid "City:"
-msgstr "Stadt:"
+#: ../plenigo_plugin/settings/SettingWooOrderTitle.php:41
+msgid "Order Title Format"
+msgstr "Bestellen Titelformat"
 
-#: plenigo_plugin/PlenigoShortcodeManager.php:401
-msgid "Country:"
-msgstr "Land:"
+#: ../plenigo_plugin/settings/SettingWooOrderTitle.php:60
+msgid "Order Title Format: replacement variables"
+msgstr "Bestellen Titelformat : Ersatzvariablen"
 
-#: plenigo_plugin/PlenigoShortcodeManager.php:402
-msgid "Edit this profile at plenigo"
-msgstr "Profil bei plenigo bearbeiten"
+#: ../plenigo_plugin/settings/SettingUseWPLogin.php:86
+msgid "Show the Word   press login form"
```

Comment:
Is this key correct? "Show the Word   press login form" i mean the space in wordpress